### DISCUSSION TITLE
Still can't get copilot to work right with blink

### DISCUSSION THREAD

**Discussion by OneOfOne** [LazyVim/LazyVim#4803] [Q&A] [answered]
It doesn't seem to complete most of the time.

**Comment by baesslerpa** [comment]
i have the same issue. after adding blink through extras everything is working except copilot

**Comment by folke** [comment, marked as answer]
Remove any customizations you did to copilot / blink / `<tab>` key anywhere / snippets.

I also recommend setting `vim.g.ai_cmp = false` to disable showing copilot completions in blink. 
Copilot suggestions will be shown as ghost text and can be inserted with `<tab>`.

It took some time to get used to this workflow, but I much prefer it over the old way in the meantime.

**Reply by baesslerpa** [reply]
@folke thx, that helped 👯
